packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/container.tsx

```diff
@@ -222,6 +230,36 @@ const CompleteResetPasswordContainer = ({
     }
   };
 
+  const maybeNotifyOAuthClient = async (accountResetData: AccountResetData) => {
```

Comment:
Maybe we could use a comment somewhere around these changes? I think it's worth noting that on this page we login oauth sync users but for other oauth users, they're logged in on CTA click of the next page. (I don't think you need to say all that in particular, just some note for future spelunking that might make it easier to see what's going on at a glance / that would have helped you!)

Reply:
+1 to this, it would be nice to have a small note to explain the purpose of this message because it can be hard someone unfamiliar to this part of the code to understand/untangle the different integrations.

Thanks for you work on this!! Looking forward to improved experience on mobile 🙌

Reply:
Thank you! I've added a comment which I hope explains the "why" part sufficient.